armi/physics/neutronics/tests/test_crossSectionManager.py

```diff
@@ -844,22 +906,37 @@ def test_createRepresentativeBlocksUsingExistingBlocks(self):
         self.assertEqual(origXSIDsFromNew["BA"], "AA")
 
     def test_interactBOL(self):
-        """Test `BOL` lattice physics update frequency."""
+        """Test `BOL` lattice physics update frequency.
+
+        .. test:: The XSGM frequency depends on the LPI frequency at BOL.
+            :id: T_ARMI_XSGM_FREQ0
+            :tests: R_ARMI_XSGM_FREQ
+        """
         self.blockList[0].r.p.timeNode = 0
         self.csm.cs[CONF_LATTICE_PHYSICS_FREQUENCY] = "BOL"
         self.csm.interactBOL()
         self.assertTrue(self.csm.representativeBlocks)
 
     def test_interactBOC(self):
-        """Test `BOC` lattice physics update frequency."""
+        """Test `BOC` lattice physics update frequency.
+
+        .. test:: The XSGM frequency depends on the LPI frequency at BOC.
+            :id: T_ARMI_XSGM_FREQ1
+            :tests: R_ARMI_XSGM_FREQ
+        """
         self.blockList[0].r.p.timeNode = 0
         self.csm.cs[CONF_LATTICE_PHYSICS_FREQUENCY] = "BOC"
         self.csm.interactBOL()
         self.csm.interactBOC()
         self.assertTrue(self.csm.representativeBlocks)
 
     def test_interactEveryNode(self):
-        """Test `everyNode` lattice physics update frequency."""
+        """Test `everyNode` lattice physics update frequency.
+
+        .. test:: The XSGM frequency depends on the LPI frequency at every time node.
+            :id: T_ARMI_XSGM_FREQ2
+            :tests: R_ARMI_XSGM_FREQ
+        """
```

Comment:
Can we put this test tag on the two coupled tests below too? Thanks!

Reply:
Done